app/utils/filters.js

```diff
@@ -24,6 +25,20 @@ function trimFilterValues (filters) {
   return filters
 }
 
+function validateDateRange(filters){
+  let result = moment(filters.fromDate, 'DD/MM/YYYY').isAfter(moment(filters.toDate, 'DD/MM/YYYY'))? 1: -1;
+  var isInvalid = false
```

Comment:
Can we change these to:
const result = ...
let isInvalid = ...

Reply:
Done, thanks